src/azure-cli/requirements.py3.windows.txt

```diff
@@ -33,7 +33,7 @@ azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==9.1.0
 azure-mgmt-containerregistry==8.2.0
 azure-mgmt-containerservice==19.1.0
-azure-mgmt-core==1.3.0
+azure-mgmt-core==1.3.1
```

Comment:
We shouldn't be updating other SDKs, any reason for this change?

Reply:
I had to make this change in order for `azdev setup` to run: the new version of `azure-mgmt-web` depends on this new version of `azure-mgmt-core`

Reply:
@zhoxing-ms can you comment on this?

Reply:
> @zhoxing-ms will I need to test the impact of the `azure-mgmt-core` on other command modules? For what it's worth, the [change log for version 1.3.1 ](https://pypi.org/project/azure-mgmt-core/)has only one entry (so it seems to be a minor update)

@StrawnSC if this is needed, please have the core team handle this change in a separte PR & make this change dependent on that. Our team doesn't own this SDK & hence we shouldn't be making these changes or be responsible for testing & handling issues that might occur due to this update.